rllib/agents/impala/vtrace_torch_policy.py

```diff
@@ -209,9 +208,13 @@ def make_time_major(policy, seq_lens, tensor, drop_last=False):
         T = tensor.shape[0] // B
     else:
         # Important: chop the tensor into batches at known episode cut
-        # boundaries. TODO(ekl) this is kind of a hack
+        # boundaries.
         T = policy.config["rollout_fragment_length"]
         B = tensor.shape[0] // T
+        # Cover cases, where we send a (small) test batch through this loss
+        # function.
+        if B == 0:
```

Comment:
Can we retain compatibility here by sending a bigger batch?

Reply:
ok, yeah, maybe the test batch should be large anyways.

Reply:
done, removed.

Reply:
made test batch large enough (32).